content/javascript/concepts/window-prompt/window-prompt.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'prompt()'
```

Comment:
The folder/file name needs to match this (`promt`)

Also, the window concept isn't merged yet but when it does, this can follow shortly after. Just make sure to create a window folder so the whole path for this changed file looks like `content/javascript/concepts/window/terms/prompt/prompt`